specs/phase0/beacon-chain.md

```diff
@@ -1512,6 +1514,7 @@ def get_attestation_deltas(state: BeaconState) -> Tuple[Sequence[Gwei], Sequence
 
 ```python
 def process_rewards_and_penalties(state: BeaconState) -> None:
+    # No rewards are applied at end of `GENESIS_EPOCH` because rewards are for work done in the previous epoch
```

Comment:
Ridiculously nitpicking 
```suggestion
    # No rewards are applied at the end of `GENESIS_EPOCH` because rewards are for work done in the previous epoch
```